<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_IdentParam</class>
 <widget class="QWidget" name="Dlg_IdentParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dlg_IdentParam</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>30</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>识别时间(ms):</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>司乘人员定位:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>摩托车牌识别:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>开启阴影消除:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>车身颜色识别:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>本地车牌占80%以上:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>默认代码:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="chkall">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>默认省份:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="edttimecontrol">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="chkvehdanger">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>危化品车辆识别:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>全选:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>驾驶员开车打电话:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>驾驶员未寄安全带:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="chklocalplate">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="chkrecogmt">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="chkshadowremove">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_15">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>副驾未寄安全带:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>遮阳板遮挡人脸:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="cmbdefaultprovince">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>京</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>津</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>沪</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>渝</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>冀</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>晋</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>辽</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>吉</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>黑</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>苏</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>浙</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>皖</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>闽</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>赣</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>鲁</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>豫</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>鄂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>湘</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>粤</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>琼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>川</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>黔</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>贵</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>滇</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>云</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>陕</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>秦</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>甘</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>陇</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>青</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>藏</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>桂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蒙</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>宁</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>新</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>澳</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>车辆品牌识别:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="chkvehlogorecog">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QCheckBox" name="chkvehlogorecog1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>车辆子品牌识别:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>车辆年款识别:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>识别标准:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QCheckBox" name="chkvehlogorecog2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QComboBox" name="cmbrecstd">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>企业标准</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>国标</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="cmbchdefaultarea">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>22</height>
             </size>
            </property>
            <property name="toolTipDuration">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>C</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>D</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>E</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>G</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>H</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>I</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>J</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>K</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>L</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>M</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>N</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>O</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>P</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Q</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>R</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>S</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>T</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>U</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>V</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>W</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Z</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkareaexcept">
            <property name="text">
             <string>除此之外</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="chkvehcolorrecog">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QCheckBox" name="chkfacelocate">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QCheckBox" name="chkphonecheck">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QCheckBox" name="chkseatbeltcheck">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QCheckBox" name="chkcopilotcheck">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QCheckBox" name="chksunvisorcheck">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>245</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>28</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>124</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
